src/forms/reset-password-popup/reset-password/messages.js

```diff
@@ -7,7 +7,7 @@ const messages = defineMessages({
     defaultMessage: 'Please check your responses and try again.',
     description: 'Error message for reset password page',
   },
-  resetPassowrdSuccess: {
+  resetPasswordSuccess: {
```

Comment:
did we fix it in the component as well ?

Reply:
yes